spec/models/tag_spec.rb

```diff
@@ -32,4 +32,27 @@
     tag = create(:tag)
     expect(build(:tag, oligo: tag.oligo)).to_not be_valid
   end
+
+  it 'can have no tag taggables' do
+    tag = create(:tag_with_taggables, taggables_count: 0)
+    expect(tag.tag_taggables).to be_empty
+  end
+
+  it 'can have many tag taggables' do
+    numTaggables = 3
+    tag = create(:tag_with_taggables, taggables_count: 3)
+    expect(tag.tag_taggables.count).to eq(3)
```

Comment:
I'm guessing you meant to use the variable above both here and on the line above?  Either way, if you do, you need to use snake_case and if you don't the variable needs removing.